packages/components/package.json

```diff
@@ -29,8 +29,8 @@
     "deploy": "npm publish --tag beta",
     "test": "../../node_modules/jest/bin/jest.js --config ../../config/jest.config.js --testPathPattern packages/components",
     "test:watch": "../../node_modules/jest/bin/jest.js --config ../../config/jest.config.js --testPathPattern packages/components --watch",
-    "test:cjs": "../../node_modules/jest/bin/jest.js --config ./config/jest.cjs.config.js --testPathPattern packages/components",
-    "test:umd": "../../node_modules/jest/bin/jest.js --config ./config/jest.umd.config.js --testPathPattern packages/components"
+    "test:cjs": "npm run build && ../../node_modules/jest/bin/jest.js --config ../../config/jest.cjs.config.js --testPathPattern packages/components",
+    "test:umd": "npm run build && ../../node_modules/jest/bin/jest.js --config ../../config/jest.umd.config.js --testPathPattern packages/components"
```

Comment:
Should we be using `npx jest` here instead of `../../node_modules/jest/bin/jest.js`?